spec/features/projects/viewing_project_spec.rb

```diff
@@ -45,6 +45,61 @@
     end
   end
 
+  context 'temporal label' do
+    context 'when a date range has been selected' do
+      before :all do
+        Capybara.reset_sessions!
+        load_page :search, project: ['C14758250-LPDAAC_ECS', 'C1000000000-LANCEAMSR2']
+        login
+        click_link "Temporal"
+        fill_in "Start", with: "2010-01-01 00:00:00\t\t"
+        fill_in "End", with: "2014-02-01 00:00:00\t\t"
+        wait_for_xhr
+        click_button('Apply Filter')
+        sleep 1
```

Comment:
We should take care of the underlying issue here, we can't let `sleeps` make it into the baseline.